msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md

```diff
@@ -0,0 +1,313 @@
+
+To enable more seamless collaboration, Microsoft created PowerPoint Live, which has become instrumental to how people work in Teams. Presenters can annotate over slides for everyone to see, using pens, highlighters, and laser pointers to draw attention to key concepts. Using Live Share canvas, your app can bring the power of PowerPoint Live inking tools with minimal effort.
+
+## Install
+
+To add the latest version of the SDK to your application using npm:
+
+```bash
+npm install @microsoft/live-share --save
+npm install @microsoft/live-share-canvas --save
+```
+
+OR
+
+To add the latest version of the SDK to your application using [Yarn](https://yarnpkg.com/):
+
+```bash
+yarn add @microsoft/live-share
+yarn add @microsoft/live-share-canvas
+```
+
+## Setting up the package
+
+Live Share canvas has two primary classes that enable turn-key collaboration: `InkingManager` and `LiveCanvas`. `InkingManager` is responsible for attaching a fully-featured `<canvas>` element to your app, while `LiveCanvas` manages the remote synchronization with other meeting participants. Used together, your app can have complete whiteboard-like functionality in just a few lines of code.
+
+| Classes                                                                     | Description                                                                                                                                                                                                                                               |
+| --------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [InkingManager](/javascript/api/@microsoft/live-share-canvas/inkingmanager) | Class that attaches a `<canvas>` element to a given `<div>` to automatically manage pen/highlighter strokes, laser pointer, lines & arrows, erasers, and cursors. Exposes a set of APIs to control which tool is active and basic configuration settings. |
+| [LiveCanvas](/javascript/api/@microsoft/live-share-canvas/livecanvas)       | A `SharedObject` class that synchronizes strokes & cursor positions from `InkingManager` for everyone in a Live Share session.                                                                                                                            |
+
+Example:
+
+```html
+<body>
+  <div id="canvas-host"></div>
+</body>
+```
+
+# [JavaScript](#tab/javascript)
+
+```javascript
+import * as microsoftTeams from "@microsoft/teams-js";
```

Comment:
I don't think this is actually necessary. I added a reference to the Teams client SDK to the demo as I was trying to get the user's display name (and picture) for live cursors, but ended borrowing your random user generator.

Reply:
@dclaux this is actually the way Steve changed the SDK. `joinContainer` is now in teams-js